sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConverter.cs

```diff
@@ -20,19 +21,7 @@ namespace Azure.Messaging.ServiceBus.Amqp
 {
     internal static class AmqpMessageConverter
     {
-        private const string EnqueuedTimeUtcName = "x-opt-enqueued-time";
-        private const string ScheduledEnqueueTimeUtcName = "x-opt-scheduled-enqueue-time";
-        private const string SequenceNumberName = "x-opt-sequence-number";
-        private const string EnqueueSequenceNumberName = "x-opt-enqueue-sequence-number";
-        private const string LockedUntilName = "x-opt-locked-until";
-        private const string PartitionKeyName = "x-opt-partition-key";
-        private const string PartitionIdName = "x-opt-partition-id";
-        private const string ViaPartitionKeyName = "x-opt-via-partition-key";
-        private const string DeadLetterSourceName = "x-opt-deadletter-source";
-        private const string TimeSpanName = AmqpConstants.Vendor + ":timespan";
-        private const string UriName = AmqpConstants.Vendor + ":uri";
-        private const string DateTimeOffsetName = AmqpConstants.Vendor + ":datetime-offset";
-        private const int GuidSize = 16;
+        internal const int GuidSize = 16;
```

Comment:
nit:  This would benefit from defining the units _(bytes, I assume?)_ and potentially mentioning what the representation is.  Is this for `System.Guid` or a specific formatted string representation?

Reply:
this gets translated into System.Guid.